### ISSUE TITLE
amp-live-list polls even when disabled

### ISSUE THREAD

**Issue by ags1773** [ampproject/amphtml#29488]
## What's the issue?

The `<amp-live-list>` component sends polling requests even when it has the `disabled` attribute

## How do we reproduce the issue?

Production live blog > https://www.barandbench.com/amp/story/news%2Flitigation%2Fsupreme-court-hears-case-concerning-adjusted-gross-revenue-agr-payable-by-telecom-companies-live-updates

1. Open above link
2. check network tab

## Which AMP version is affected?
amp version > 2007102309001

![Screenshot 2020-07-24 at 9 25 36 AM](https://user-images.githubusercontent.com/27222520/88359585-d02fce00-cd90-11ea-9d21-2cde47111bff.png)
![Screenshot 2020-07-24 at 9 34 26 AM](https://user-images.githubusercontent.com/27222520/88359622-efc6f680-cd90-11ea-9a4f-f282d32e6675.png)

**Comment by caroqliu** [comment]
Thanks for filing @ags1773. I can reproduce the issue and will look into it. :)

**Comment by caroqliu** [comment]
Looks like the polling restarts when the document is visible again (i.e. if a user goes to another tab and comes back): 
https://github.com/ampproject/amphtml/blob/0d15009319dc7ea5758e6e0b09d78d96570063b7/extensions/amp-live-list/0.1/live-list-manager.js#L302-L305

This bug should be fixed by also checking `isEnabled` as is done when the instance is first registered:
https://github.com/ampproject/amphtml/blob/0d15009319dc7ea5758e6e0b09d78d96570063b7/extensions/amp-live-list/0.1/live-list-manager.js#L281
